<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="stylesheet" href="../extension/bulma/css/bulma.min.css">
  <link rel="stylesheet" href="../extension/bulma/bulma.sass">
  <title>Description</title>
  <script src="https://cdn.staticfile.org/jquery/2.1.1/jquery.min.js"></script>
  <script src="https://kit.fontawesome.com/2cc9a2c7f9.js" crossorigin="anonymous"></script>
  <script src="https://cdn.jsdelivr.net/gh/cferdinandi/smooth-scroll@15/dist/smooth-scroll.polyfills.min.js"></script>
  <link rel="stylesheet" href="../css/description.css">
</head>

<body>
  <nav class="navbar visable is-black role=" navigation"
    style="top: -60px;z-index: 101;transition: transform 0.2s ease-in-out, opacity 0.2s ease-in-out;position: fixed;width: 100%;">
    <span style="margin: 0 auto;">
      <div class="navbar-menu" id="navbarmenu">
        <a href="../index.html" class="navbar-item">
          <img src="../img/index/nav-header.png" height="40">
        </a>
        <a class="navbar-item" href="../index.html">Home</a>
        <div class="navbar-item has-dropdown is-hoverable">
          <a class="navbar-link is-arrowless">Project</a>
          <div class="navbar-dropdown">
            <a class="navbar-item" href="./description.html">Description</a>
            <a class="navbar-item" href="#">Design</a>
            <a class="navbar-item" href="#">Engineering</a>
            <a class="navbar-item" href="#">Implementation</a>
          </div>
        </div>
        <a class="navbar-item">Parts</a>
        <a class="navbar-item">Model</a>
        <a class="navbar-item">Human Practice</a>
        <a class="navbar-item" href="team.html">Team</a>
        <a class="navbar-item">Judging</a>
      </div>
    </span>
    <!-- 入口 -->

  </nav>
  <div style="vertical-align: middle;background-image: url(../img/description/T--TAS_Taipei--Description_big.jpg);">
    <div class="intro-1">DESCRIPTION</div>
    <div class="intro-2">DESCRIPTION</div>
    <script>
      new SmoothScroll('a[href*="#"]', {
        easing: 'easeInOutQuart'
      });
    </script>
  </div>
  <progress class="progress" value="0" max="100"
    style="width: 100px;height:12px;transform-origin: top left;transform:rotate(90deg);position: fixed;left: 50px; "></progress>

  <div class="columns">
    <div class="column is-3 index-column" style="margin: 60px;">
      <b onclick="$('html, body').animate({scrollTop:0}, 'slow')" href="#"
        style="cursor: pointer;margin-top: 50px;">DESCRIPTION</b>
      <ul class="title-nav">
        <li class="active">
          <div></div>
          <a href="#title1">BACKGROUND</a>
        </li>
        <li>
          <div></div>
          <a href="#title2">CURRENT SOLUTIONS</a>
        </li>
        <li>
          <div></div>
          <a href="#title3">OUR GOAL</a>
        </li>
        <li>
          <div></div>
          <a href="#title4">THE BIGGER PICTURE</a>
        </li>
        <li>
          <div></div>
          <a href="#title5">REFERENCES</a>
        </li>
      </ul>
    </div>
    <div class="column is-7">
      <div class="content">
        <div class="paragragh">
          <h1 id="title1">Background</h1>
          <h2>What are Viruses?</h2>
          <p>Viruses are microbes that infect cells through modes of endocytic mechanisms or direct fusion to the plasma
            membrane (Cohen, 2016).<sup><a href="#cit1">[1]</a></sup> Upon cell invasion, viruses exploit the cell’s
            cellular process to create new virus
            particles through replication of its genetic information and production of viral proteins (Baron et al.,
            1996). They proceed to disseminate to target organs which may be sufficient to cause disease and symptoms
            and sites that release them back into the environment (Baron et al., 1996).</p>
          <h2>Virus Related Infection and Deaths</h2>
          <figure>
            <img src="../img/description/T--TAS_Taipei--Description1.png">
            <figcaption>
              Figure 1: Influenza Cases Worldwide March 2020 (WHO | Influenza Update - 362, n.d.).
            </figcaption>
          </figure>
          <p>Seasonal flus account for approximately 3 to 5 million cases and 250 to 500 thousands deaths worldwide
            annually (Influenza (Seasonal), n.d.). Pandemics such as the ongoing SARS-CoV2 pandemic has already caused
            30.9 million cases and 958 thousand deaths, and the numbers continue to rise (WHO Coronavirus Disease
            (COVID-19) Dashboard, n.d.). <b>The great quantity of people subjected to these diseases makes diagnostic
              measures crucial to installing preventive and therapeutic action.</b></p>
          <figure>
            <img src="../img/description/T--TAS_Taipei--Description2.png">
            <figcaption>
              Figure 2: Worldwide SARS-CoV-2 Cases Up to Oct 23rd, 2020 (Microsoft Bing COVID-19 Tracker, n.d.).
            </figcaption>
          </figure>
        </div>
        <div class="paragragh">
          <h1 id="title2">Current Solutions</h1>
          <h2>Sample Collection and Purification</h2>
          <p>In order to test for a viral infection, a sample must be collected and at times isolated. Currently, there
            are two main methods for viral collection: through nasopharyngeal swabs or saliva. For nasopharyngeal swabs,
            a swab is inserted into the nostril until it reaches the pharynx region. It is left there for a few seconds
            to absorb secretions and removed while slowly rotating to obtain cells (Marty et al., 2020). This is usually
            done by a medical personnel. Saliva on the other hand is usually self
            collected by spitting into a sterile
            container with saline (Jamal et al., n.d.). Various articles have shown that saliva collection is highly
            sensitive and yields similar test outcomes, making it a noninvasive reliable alternative to swabs
            (Greenwood, 2020; Iwasaki et al., 2020; Jamal et al., n.d.; Nasopharyngeal Swab vs. Saliva for COVID-19
            Diagnosis, 2020).</p>
          <figure>
            <img src="../img/description/T--TAS_Taipei--Description3.jpeg" />
            <figcaption>Figure 3: Nasopharyngeal Swab from Miraclean Technology (Nasopharyngeal Sampling Swabs, n.d.).
            </figcaption>
          </figure>
          <p>Most viral extraction or purification techniques center around commercial RNA extraction kits. They usually
            involve using lysis buffers, wash buffers, elution buffers, membrane columns, and centrifugation to
            ultimately extract in a lengthy process (Tan & Yiap, 2009). These are more often for nucleic acid tests and
            aren’t usually required in rapid test settings.
          </p>
        </div>
        <div class="paragragh">
          <h1 id="title3">Our Goal</h1>
          <h2>Theory</h2>
          <figure>
            <img src="../img/description/T--TAS_Taipei--Description5.gif" />
            <figcaption>Figure 5: The rolling circle amplification RCA process on a molecular level.</figcaption>
          </figure>
          <p>To address some of the issues with the aforementioned viral detection tests. We employed an isothermal
            methodology known as padlock probe rolling circle amplification. The two ends of the padlock probe are
            designed to bind to a specific complementary target sequence. Upon hybridization, a ligase, usually T4
            ligase, can ligate the ends of the padlock probe due to a 5’ phosphorylation on the padlock probe and the
            presence of ATP as a cofactor (Hamidi & Ghourchian, 2015). The result is a circularized padlock probe, which
            can now act as a template for DNA amplification (Hamidi & Perreault, 2019). If the padlock probe is not
            complementary to the target sequence, ligation should not occur (Gu et al., 2018). Without a circularized
            padlock probe, continual DNA amplification will not be able to occur and thus the readout will show a
            negative result.<br>

            The subsequent methods of amplification varies. Some involve the use of the 3’ to 5’ exonuclease activity of
            phi29 DNA polymerase to cut the target sequence until the ligation junction is reached, which enables DNA
            synthesis to proceed (Li et al., 2017). Others use an exonuclease or RNase to achieve the same function or
            direct annealing of a forward primer to the circularized backbone (Joffroy et al., 2018; RNase H-Assisted
            RNA-Primed Rolling Circle Amplification for Targeted RNA Sequence Detection | Scientific Reports, n.d.).
            Whatever the case, the DNA polymerase of choice, usually Bst DNA polymerase or phi29 DNA polymerase, has an
            existing DNA strand to synthesize DNA strands in circles around the padlock probe through its helicase
            activity (Joffroy et al., 2018). A second reverse primer that binds to the newly synthesized DNA strands is
            sometimes used to further increase DNA synthesis by introducing a branching-like amplification (Hamidi &
            Ghourchian, 2015, p. 1; Hamidi & Perreault, 2019, 2019; Larsson et al., 2004). Finally, the readout usually
            uses fluorescent probes that recognize the synthesized strand and is thus quantifiable (Hamidi & Ghourchian,
            2015, p. 1; Larsson et al., 2004).
          </p>
        </div>
        <div class="paragragh">
          <h1 id="title4">The Bigger Picture</h1>
          <p>Our RCA colorimetric experiments ultimately serve as the viral diagnosis portion of our project. But to
            truly bring it to life, we have to incorporate portions such as our modeling hardware piece, where we
            presented a theory and 2 in 1 saliva collection and purification device design to allow nucleic acid
            isolation. Using our software to quantify our qualitative color readouts into pH values and inputting the
            rate of pH change into our model, we determined the RNA concentration of our reactions. With RNA
            concentration, the viral load of the patient can be reported. Although viral load does not correlate with
            the severity of the viral disease, it can be a good indicator of the time since viral exposure and the level
            of infectivity of a sick individual. From a human practices standpoint, this can present ways for
            individuals to seek immediate medical attention for the benefit of the individual and population. This
            ultimately allows us to implement our project as a viral diagnostic home test kit that is not only highly
            usable, but provides useful information and helpful implications to anyone being tested.</p>
        </div>
        <div class="paragragh">
          <h1 id="title5">References</h1>
          <ol>
            <li id="cit1">Alhassan, A., Li, Z., Poole, C. B., & Carlow, C. K. S. (2015). Expanding the MDx toolbox for
              filarial diagnosis and surveillance. Trends in Parasitology, 31(8), 391–400.
              https://doi.org/10.1016/j.pt.2015.04.006</li>
            <li id="cit2">Baron, S., Fons, M., & Albrecht, T. (1996). Viral Pathogenesis. In Medical Microbiology. 4th
              edition. University of Texas Medical Branch at Galveston. https://www.ncbi.nlm.nih.gov/books/NBK8149/</li>
            <li id="cit3">Bhagavan, N. V., & Ha, C.-E. (2015). Chapter 22—DNA Replication, Repair, and Mutagenesis. In
              N. V. Bhagavan & C.-E. Ha (Eds.), Essentials of Medical Biochemistry (Second Edition) (pp. 401–417).
              Academic Press. https://doi.org/10.1016/B978-0-12-416687-5.00022-1</li>
            <li id="cit4">Biolabs, N. E. (n.d.). Loop-Mediated Isothermal Amplification | NEB. Retrieved September 21,
              2020, from
              https://www.neb.com/applications/dna-amplification-pcr-and-qpcr/isothermal-amplification/loop-mediated-isothermal-amplification-lamp
            </li>
            <li id="cit5">CDC. (2020, February 11). Information for Laboratories about Coronavirus (COVID-19). Centers
              for Disease Control and Prevention.
              https://www.cdc.gov/coronavirus/2019-ncov/lab/grows-virus-cell-culture.html</li>
            <li id="cit6">Cohen, F. S. (2016). How Viruses Invade Cells. Biophysical Journal, 110(5), 1028.
              https://doi.org/10.1016/j.bpj.2016.02.006</li>
            <li id="cit7">False-Negative Rate of RT-PCR SARS-CoV-2 Tests. (n.d.). American College of Cardiology.
              Retrieved June 5, 2020, from
              http%3a%2f%2fwww.acc.org%2flatest-in-cardiology%2fjournal-scans%2f2020%2f05%2f18%2f13%2f42%2fvariation-in-false-negative-rate-of-reverse
            </li>
            <li id="cit8">Ge, Y., Zhou, Q., Zhao, K., Chi, Y., Liu, B., Min, X., Shi, Z., Zou, B., & Cui, L. (2017).
              Detection of influenza viruses by coupling multiplex reverse-transcription loop-mediated isothermal
              amplification with cascade invasive reaction using nanoparticles as a sensor. International Journal of
              Nanomedicine, Volume 12, 2645–2656. https://doi.org/10.2147/IJN.S132670</li>
            <li id="cit9">Gordon, J., & Michel, G. (2008). Analytical Sensitivity Limits for Lateral Flow Immunoassays.
              Clinical Chemistry, 54(7), 1250–1251. https://doi.org/10.1373/clinchem.2007.102491</li>
            <li id="cit10">Greenwood, M. (2020, August 15). Yale’s rapid COVID-19 saliva test receives FDA emergency use
              authorization. YaleNews.
              https://news.yale.edu/2020/08/15/yales-rapid-covid-19-saliva-test-receives-fda-emergency-use-authorization
            </li>
            <li id="cit11">Gu, L., Yan, W., Liu, L., Wang, S., Zhang, X., & Lyu, M. (2018). Research Progress on Rolling
              Circle Amplification (RCA)-Based Biomedical Sensing. Pharmaceuticals, 11(2), 35.
              https://doi.org/10.3390/ph11020035</li>
            <li id="cit12">Hamidi, S. V., & Ghourchian, H. (2015). Colorimetric monitoring of rolling circle
              amplification for detection of H5N1 influenza virus using metal indicator. Biosensors and Bioelectronics,
              72, 121–126. https://doi.org/10.1016/j.bios.2015.04.078</li>
            <li id="cit13">Hamidi, S. V., & Perreault, J. (2019). Simple rolling circle amplification colorimetric assay
              based on pH for target DNA detection. Talanta, 201, 419–425. https://doi.org/10.1016/j.talanta.2019.04.003
            </li>
            <li id="cit14">Harcourt, J., Tamin, A., Lu, X., Kamili, S., Sakthivel, S. K., Murray, J., Queen, K., Tao,
              Y., Paden, C. R., Zhang, J., Li, Y., Uehara, A., Wang, H., Goldsmith, C., Bullock, H. A., Wang, L.,
              Whitaker, B., Lynch, B., Gautam, R., … Thornburg, N. J. (n.d.). Severe Acute Respiratory Syndrome
              Coronavirus 2 from Patient with Coronavirus Disease, United States—Volume 26, Number 6—June 2020—Emerging
              Infectious Diseases journal—CDC. https://doi.org/10.3201/eid2606.200516</li>
            <li id="cit15">Hematian, A., Sadeghifard, N., Mohebi, R., Taherikalani, M., Nasrolahi, A., Amraei, M., &
              Ghafourian, S. (2016). Traditional and Modern Cell Culture in Virus Diagnosis. Osong Public Health and
              Research Perspectives, 7(2), 77. https://doi.org/10.1016/j.phrp.2015.11.011</li>
            <li id="cit16">Influenza (Seasonal). (n.d.). Retrieved September 21, 2020, from
              https://www.who.int/news-room/fact-sheets/detail/influenza-(seasonal)</li>
            <li id="cit17">Iwasaki, S., Fujisawa, S., Nakakubo, S., Kamada, K., Yamashita, Y., Fukumoto, T., Sato, K.,
              Oguri, S., Taki, K., Senjo, H., Hayasaka, K., Sugita, J., Konno, S., Nishida, M., & Teshima, T. (2020).
              Comparison of SARS-CoV-2 detection in nasopharyngeal swab and saliva. MedRxiv, 2020.05.13.20100206.
              https://doi.org/10.1101/2020.05.13.20100206</li>
            <li id="cit18">Jamal, A. J., Mozafarihashjin, M., Coomes, E., Powis, J., Li, A. X., Paterson, A.,
              Anceva-Sami, S., Barati, S., Crowl, G., Faheem, A., Farooqi, L., Khan, S., Prost, K., Poutanen, S.,
              Taylor, M., Yip, L., Zhong, X. Z., McGeer, A. J., Mubareka, S., … Walmsley, S. (n.d.). Sensitivity of
              Nasopharyngeal Swabs and Saliva for the Detection of Severe Acute Respiratory Syndrome Coronavirus 2.
              Clinical Infectious Diseases. https://doi.org/10.1093/cid/ciaa848</li>
            <li id="cit19">Joffroy, B., Uca, Y. O., Prešern, D., Doye, J. P. K., & Schmidt, T. L. (2018). Rolling circle
              amplification shows a sinusoidal template length-dependent amplification bias. Nucleic Acids Research,
              46(2), 538–545. https://doi.org/10.1093/nar/gkx1238</li>
            <li id="cit20">Koczula, K. M., & Gallotta, A. (2016). Lateral flow assays. Essays in Biochemistry, 60(1),
              111. https://doi.org/10.1042/EBC20150012</li>
            <li id="cit21">Larsson, C., Koch, J., Nygren, A., Janssen, G., Raap, A. K., Landegren, U., & Nilsson, M.
              (2004). In situ genotyping individual DNA molecules by target-primed rolling-circle amplification of
              padlock probes. Nature Methods, 1(3), 227–232. https://doi.org/10.1038/nmeth723</li>
            <li id="cit22">Li, X.-Y., Du, Y.-C., Zhang, Y.-P., & Kong, D.-M. (2017). Dual functional Phi29 DNA
              polymerase-triggered exponential rolling circle amplification for sequence-specific detection of target
              DNA embedded in long-stranded genomic DNA. Scientific Reports, 7(1), 6263.
              https://doi.org/10.1038/s41598-017-06594-1</li>
            <li id="cit23">Lippi, G., Mattiuzzi, C., Bovo, C., & Plebani, M. (2020). Current laboratory diagnostics of
              coronavirus disease 2019 (COVID-19). Acta Bio Medica Atenei Parmensis, 91(2), 137–145.
              https://doi.org/10.23750/abm.v91i2.9548</li>
            <li id="cit24">Marty, F. M., Chen, K., & Verrill, K. A. (2020). How to Obtain a Nasopharyngeal Swab
              Specimen. New England Journal of Medicine, 382(22), e76. https://doi.org/10.1056/NEJMvcm2010260</li>
            <li id="cit25">Nasopharyngeal swab vs. Saliva for COVID-19 diagnosis. (2020, May 21). News-Medical.Net.
              https://www.news-medical.net/news/20200521/Nasopharyngeal-swab-vs-saliva-for-COVID-19-diagnosis.aspx</li>
            <li id="cit26">Rapid Influenza Diagnostic Tests | CDC. (2019, November 12).
              https://www.cdc.gov/flu/professionals/diagnosis/clinician_guidance_ridt.htm</li>
            <li id="cit27">RNase H-assisted RNA-primed rolling circle amplification for targeted RNA sequence detection
              | Scientific Reports. (n.d.). Retrieved September 22, 2020, from
              https://www.nature.com/articles/s41598-018-26132-x</li>
            <li id="cit28">Roberts, J., Bebenek, K., & Kunkel, T. (1988). The accuracy of reverse transcriptase from
              HIV-1. Science, 242(4882), 1171–1173. https://doi.org/10.1126/science.2460925</li>
            <li id="cit29">Sajid, M., Kawde, A.-N., & Daud, M. (2015). Designs, formats and applications of lateral flow
              assay: A literature review. Journal of Saudi Chemical Society, 19(6), 689–705.
              https://doi.org/10.1016/j.jscs.2014.09.001</li>
            <li id="cit30">Tan, S. C., & Yiap, B. C. (2009). DNA, RNA, and Protein Extraction: The Past and The Present.
              Journal of Biomedicine and Biotechnology, 2009, 1–10. https://doi.org/10.1155/2009/574398</li>
            <li id="cit31">Virus contaminations of cell cultures – A biotechnological view. (n.d.). Retrieved September
              21, 2020, from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3463984/</li>
            <li id="cit32">WHO Coronavirus Disease (COVID-19) Dashboard. (n.d.). Retrieved September 21, 2020, from
              https://covid19.who.int</li>
          </ol>
        </div>
      </div>
    </div>
    <div class="column is-1 index-column">
      <span class="up-arrow" onclick="$('html, body').animate({scrollTop:0}, 'slow')" style="cursor: pointer;">
        <span class="up-arrow-bg" style="margin-top: 120px;"></span>
        <img class="up-arrow-icon" src="../img/description/arrow.png"  style="margin-top: 130px;"/>
      </span>
    </div>
  </div>
  <!-- 滚动进度条 -->
  <script>
    (function () {
      let current = document.documentElement.scrollTop || document.body.scrollTop;  // 获取滚动条的高度
      let pageHeight = document.body.scrollHeight || document.documentElement.scrollHeight; // 页面总高度
      let windowHeight = document.documentElement.clientHeight || document.body.clientHeight; // 浏览器视口高度
      let scrollAvail = pageHeight - windowHeight; // 可滚动的高度
      var timeoutId;
      console.log('可滚动的高度：', scrollAvail);
      window.onscroll = function () {
        let scrollTop = document.documentElement.scrollTop || document.body.scrollTop;  // 获取滚动条的高度
        console.log('滚动条的高度：', scrollTop);
        $('.progress').val(scrollTop / scrollAvail * 100);    // 计算占比
        $('.progress').fadeIn();
        if (timeoutId !== undefined) {
          clearTimeout(timeoutId);
        }
        timeoutId = setTimeout(() => {
          $('.progress').fadeOut()
        }, 2000);
        // 顶部栏动态
        if (scrollTop > current) {
          $('.navbar').removeClass('visable');
        } else {
          $('.navbar').addClass('visable');
        }
        current = scrollTop;
      };
      window.onload = function () {
        console.log('load')
        $('.progress').css({
          position: 'fixed',
          width: $(window).height() / 3,
          top: ($(window).height() - $(window).height() / 3) / 2,
        }).fadeOut();
      };
      window.onloadstart = function () {
        console.log('start')
        $('.progress').css({
          position: 'fixed',
          width: $(window).height() / 3,
          top: ($(window).height() - $(window).height() / 3) / 2,
        }).fadeOut();
      }
      window.onresize = function () {
        $('.progress').css({
          position: 'fixed',
          width: $(window).height() / 3,
          top: ($(window).height() - $(window).height() / 3) / 2,
        }).fadeIn();
        setTimeout(() => {
          $('.progress').fadeOut()
        }, 3000)
      };
    }());

  </script>
  <!-- 锚点定位 -->
  <script>
    var $navs = $('.title-nav li'),          // 导航
      $sections = $('.content div'),       // 模块
      $window = $(window),
      navLength = $navs.length - 1;

    $window.on('scroll', function () {
      var scrollTop = $window.scrollTop(),
        len = navLength;

      for (; len > -1; len--) {
        var that = $sections.eq(len);
        if (scrollTop >= that.offset().top) {
          $navs.removeClass('active').eq(len).addClass('active');
          break;
        }
      }
    });
  </script>
</body>


</html>